javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentClassLoader.java

```diff
@@ -488,7 +488,7 @@ protected Class<?> loadClass(String name, boolean resolve) throws ClassNotFoundE
       // prometheus exporter uses jdk http server, load it from the platform class loader
       // some custom extensions use java.sql classes, make these available to agent and extensions
```

Comment:
should we update this comment as well? something like
```suggestion
      // some custom extensions use java.* classes, make these available to agent and extensions
```

Reply:
I don't like the proposed wording because most of the `java` classes can obviously be used just fine. The problem is with only some `java` classes like `java.sql` that is not in boot loader on recent java sdks.

Reply:
how about

```suggestion
      // some custom extensions use java.* classes which are not in the boot loader such as java.sql.* and java.net.http.*
```